review_july (#42)
fiskaltrust/rollout-doc: i18n/de/middleware/step-by-step/6-data-center-as-operational-environment/README.md

# Rechenzentrum als Operational Environment

> Dieser Leitfaden führt Sie durch ein Setup wo Ihre Outlets Kassenterminals verwenden und durch ein zentrales Kassensystem in Ihrem Rechenzentrum verrechnen. Weiters führt das Kassensystem für jedes Outlet eine eigene CashBox (mit eigener Queue und fiskaltrust.SCU), welche jeweils abhängig davon von welchem Outlet die Anfrage gekommen ist, angesprochen wird.

Reviewer: Sehr komplizierter Satzbau

1. eine SCU
2. eine Queue

> 3. eine CashBox, welche die fiskaltrust.SCU  und die CashBox beinhaltet

Reviewer: ```suggestion
3. eine CashBox, welche die fiskaltrust.SCU und die Queue beinhaltet
```

## 2 - Erstellung der fiskaltrust.SCU 
### Schritt 1 - Navigation im Menü

> Wählen Sie linkerhand im Menü `Konfiguration` - `TSE / Signatur-Erstellungs-Einheit`.

Reviewer: ```suggestion
Wählen Sie im Menü `Konfiguration` - `TSE / Signatur-Erstellungs-Einheit`.
```

![scu_add](../images/scu-add.png)

### Schritt 3 - Eingabe der Details

> Geben Sie einen aussagekräftigen Namen für Ihre neue fiskaltrust.SCU  im Feld Beschreibung an, wählen Sie das korrekte Paket und die neueste Version für Ihre konkrete TSE und wählen Sie das Outlet, in welchem diese SCU aktiv sein soll. Nach Auswahl vom Paket wird das fiskaltrust.Portal automatisch die aktuellste Paketversion als Standard auswählen.

Reviewer: ```suggestion
Geben Sie einen aussagekräftigen Namen für Ihre neue fiskaltrust.SCU im Feld `Beschreibung` an, wählen Sie das korrekte Paket und die neueste Version für Ihre konkrete TSE und wählen Sie das Outlet, in welchem diese SCU aktiv sein soll. Nach Auswahl des Pakets wird automatisch die aktuellste Version verwendet.
```

![main](../images/main-scu.png)

> Klicken Sie `Speichern` sobald Sie alle Informationen eingegeben habe. Danach werden Sie zu einem Schirm umgeleitet, wo Sie zusätzliche technische Details konfigurieren können. Die Mehrheit dieser Einstellungen ist TSE-spezifisch (*im folgenden Beispiel haben wireine fiskaly cloudbasierende TSE * ausgewählt) und erfordert die korrekten Werte für diese TSE (z.B. den richtigen Laufwerksbuchstaben oder den korrekten API Schlüssel).

Reviewer: ```suggestion
Klicken Sie `Speichern` sobald Sie alle Informationen eingegeben haben. Im folgenden Schritt können Sie zusätzliche technische Details konfigurieren. Die Mehrheit dieser Einstellungen ist TSE-spezifisch (*im folgenden Beispiel wurde eine fiskaly cloudbasierende TSE * ausgewählt) und erfordert die korrekten Werte für diese TSE (z.B. den richtigen Laufwerksbuchstaben oder den korrekten API Schlüssel).
```

> ### Step 3 - Entering the details

Reviewer: ```suggestion
### Schritt 3 - Details angeben
```

> Geben Sie einen aussagekräftigen Namen für Ihre neue Queue im Feld Beschreibung an, wählen Sie `fiskaltrust.Middleware.Queue.MySQL` als gewünschten Datenbanktyp, sowie die neueste Version, wählen Sie den gewünschten Timeout Wert aus (Standard sind 15.000 Millisekunden), geben Sie die Serien- oder Identifikationsnummer Ihrer Kasse an (*oder wählen Sie `Generieren der Identifikation` aus um das fiskaltrust.Portal eine eindeutige Nummer generieren zu lassen*) und wählen Sie das Outlet, in welchem diese Queue aktiv ist aus.

Reviewer: ```suggestion
Geben Sie einen aussagekräftigen Namen für Ihre neue Queue im Feld `Beschreibung` an, wählen Sie `fiskaltrust.Middleware.Queue.MySQL` als gewünschten Datenbanktyp, sowie die neueste Version, wählen Sie den gewünschten Timeout Wert aus (Standard sind 15.000 Millisekunden), geben Sie die Serien- oder Identifikationsnummer Ihrer Kasse an (*oder wählen Sie `Generieren der Identifikation` aus um eine eindeutige Nummer zu erzeugen*) und wählen Sie das Outlet, in welchem diese Queue aktiv ist aus.
```

Reviewer: sowie die neueste Version --> Von was?

![main_queue](../images/main-queue-mysql.png)

Sobald Sie alle Informationen eingetragen haben, scrollen Sie ans untere Ende der Seite und klicken `Speichern`, worauf Sie auf einen Schirm umgeleitet werden, wo Sie zusätzliche technische Details konfigurieren können.

![](../images/queue-config-mysql.png)

> Der wichtigste Teil hier der MySQL Connection String und die URL unter welcher die Queue IPOS Requests von Ihrer Kasse entgegenehmen soll.

Reviewer: ```suggestion
Der wichtigste Teil hier der _MySQL Connection String_ und die URL unter welcher die Queue Anfragen von Ihrer Kasse entgegennehmen soll.
```

### Schritt 3 - Eingabe der Details

> Geben Sie einen aussagekräftigen Namen für Ihre neue CashBox im Feld Beschreibung an und wählen Sie das Outlet, in welchem diese CashBox aktiv sein soll aus.

Reviewer: ```suggestion
Geben Sie einen aussagekräftigen Namen für Ihre neue CashBox im Feld `Beschreibung` an und wählen Sie das Outlet, in welchem diese CashBox aktiv sein soll aus.
```

![](../images/main-cashbox.png)

> Klicken Sie auf `Speichern` sobald Sie alle Informationen eingegeben haben und die neue CashBox sollte auf dem folgenden Schirm unter den CashBoxen Ihres Kunden aufgeführt sein.

Reviewer: ```suggestion
Klicken Sie auf `Speichern` sobald Sie alle Informationen eingegeben haben. Die neue CashBox ist unter den CashBoxen Ihres Kunden aufgeführt.
```

### Schritt 4 - Verbindung der Komponenten

> Zurück bei der Liste der CashBox, klicken Sie den `Bearbeiten per Liste` Knopf.

Reviewer: ```suggestion
Zurück bei der Liste der CashBoxen, klicken Sie den `Bearbeiten per Liste` Knopf.
```

Reviewer: Gesprochene Sprache

![](../images/edit-cashbox.png)

> Auf dem folgenden Schirm müssen Sie die vorher erstellte Queue für diese CashBox auswählen, indem Sie die richtige Zeile anwählen.

Reviewer: ```suggestion
Auf dem folgenden Schirm müssen Sie die vorher erstellte Queue für diese CashBox auswählen.
```

> Wir haben jetzt die Queue für die CashBox konfiguriert und sie mit der SCU verbunden, müssen jedoch noch die SCU selbst der CashBox hinzufügen, indem wir die richtige Zeile anwählen.

Reviewer: ```suggestion
Die Queue ist für die CashBox konfiguriert und mit der SCU verbunden. Nun muss die SCU  der CashBox hinzugefügt werden.
```

## 5 - Weitere Outlets

Wiederholen Sie die Punkte 2 bis 4 für jedes weitere Outlet, welches Sie konfigurieren möchten und wo Sie Ihrem POS-System eine Queue zur Verfügung stellen möchten.

> 

Reviewer: gesprochene Sprache

> ## 6 - CashBox Build

Reviewer: ```suggestion
## 6 - CashBox Rebuild
```

Dies ist ein **wichtiger Schritt**, welchen Sie immer durchführen müssen nachdem Sie eine CashBox erstellt, **oder geändert**, haben.

> Um ein Build, oder Rebuild, einer CashBox durchzuführen, öffnen Sie die Liste aller Ihrer CashBoxen und klicken den `Rebuild configuration` Knopf bei der entsprechenden CashBox, sofern dieser in Orange markiert ist.

Reviewer: ```suggestion
Um ein Rebuild, einer CashBox durchzuführen, öffnen Sie die Liste aller Ihrer CashBoxen und klicken den `Rebuild configuration` Knopf bei der entsprechenden CashBox, sofern dieser Orange ist.
```
